docs/examples/000/1. hello-world/main.bicep

```diff
@@ -0,0 +1,11 @@
+//Sample hello world

```

Comment:
nit:

Can the name of the folder be: `01-hello-world`? spaces in filenames scare me :)